server/metadata/tenant.go

```diff
@@ -374,6 +375,19 @@ func (m *TenantManager) GetTenant(ctx context.Context, namespaceId string) (*Ten
 	return tenant, nil
 }
 
+func (m *TenantManager) AllTenants(_ context.Context) []*Tenant {
+	m.RLock()
+	defer m.RUnlock()
+
+	tenants := make([]*Tenant, len(m.tenants))
+	i := 0
+	for _, t := range m.tenants {
```

Comment:
just do `for i, t := range m.tenants {}`

Reply:
map